chip-testing/src/BridgeTestInstance.ts

```diff
@@ -22,6 +22,13 @@ import { Storage } from "@project-chip/matter.js/storage";
 import { ByteArray } from "@project-chip/matter.js/util";
 import { TestInstance } from "./GenericTestApp.js";
 
+// Note - inevitable future confusion
+//
+// See https://github.com/project-chip/connectedhomeip/issues/30467
+//
+// Bug in CHIP tests, should be fixed for 1.4 but not available here for reasons.  Meanwhile give them what they expect
+Object.assign(BridgedDeviceBasicInformation.Cluster, { revision: 2 });
```

Comment:
We should remove this, the *1_1 tessts are gone, so no need to anymore

Reply:
Oops, I actually did but I think I did a hard reset due to other changes before I committed